i18n/ja/docusaurus-plugin-content-docs/current/guides/integration-mobile-client-server.md

```diff
@@ -0,0 +1,361 @@
+:::
+
+You will need to pass the `Identity` response into the mobile app: see [Configure the UID2 Mobile SDK for your mobile app](#configure-the-uid2-mobile-sdk-for-your-mobile-app).
+
+:::note
+For security reasons, the API key and secret used in token generation must be called server-side. Do not store these values inside a mobile app. For details, see [Security of API Key and Client Secret](../getting-started/gs-credentials.md#security-of-api-key-and-client-secret).
+:::
+
+## Server-Side Token Refresh
+
+Token refresh is automatically enabled inside the UID2 mobile SDKs; you don't need to manage it explicitly on the server side.
+
+You might decide to do server-side token refresh if you want to keep your changes in the mobile apps as simple as possible.
+
+However, if you decide you want to manage token refresh on the server side and not the client/mobile side, you can do so using one of the following:
+
+- Call the [POST&nbsp;/token/refresh](../endpoints/post-token-refresh.md) endpoint.
+- Use one of the Publisher Client classes, in one of the UID2 server-side SDKs. These classes simplify the request into a single method call. 
+
+  For instructions, see [UID2 SDK for Java, Publisher Server-Only Integration section](../sdks/uid2-sdk-ref-java.md#server-only-integration) or [UID2 SDK for Python, Publisher Server-Only Integration section](../sdks/uid2-sdk-ref-python.md#server-only-integration).
+
+Then, pass the newly refreshed `Identity` value to the mobile app by following the rest of this guide.
+
+## Add the UID2 Mobile SDK into Your Mobile App
+
+For installation instructions, refer to one of the following:
+
+- [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md)
+- [UID2 SDK for iOS Reference Guide](../sdks/uid2-sdk-ref-ios.md)
+
+For example, for Android, follow the UID2 SDK documentation to:
+- Add the pre-built UID2 SDK for Android libraries into your mobile app: see [Installation](../sdks/uid2-sdk-ref-android.md#installation).
+- Learn how to instantiate the UID2Manager instance in your Android app: see [Android Initialization](../sdks/uid2-sdk-ref-android.md#android-initialization).
+ here.
+ 
+ For iOS, refer to the [Usage Guidelines](../sdks/uid2-sdk-ref-ios.md#usage-guidelines).
+
+## Using the UID2 Integration Environment
+
+By default, the SDK is configured to work with the UID2 production environment: `https://prod.uidapi.com`. If you want to use the UID2 integration environment instead, provide the following URL in your call to UID2Manager initialization:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.init(
+  context = this,
+  serverUrl = "https://operator-integ.uidapi.com"
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+// Must be set before UID2Manager.shared is accessed UID2Settings.shared.environment = .custom(
```

Comment:
_UID2Settings.shared.environment = .custom(_ should be starting on a new line

Reply:
Fixed.